spec/rack/test/cookie_spec.rb

```diff
@@ -216,4 +216,10 @@
       last_request.cookies.should == { "value" => "1" }
     end
   end
+
+  it "allow symbol access" do
+    jar = Rack::Test::CookieJar.new
+    jar["value"] = "foo;abc"
+    jar[:value].should == "foo;abc"
+  end
```

Comment:
I think we should move this closer to the other tests for `Rack::Test::CookieJar`, around line 46 would make sense. Could you fix that @Anorlondo448 and also rebase on latest `master` so we can see that Travis is still happy with this change in place?